packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 46.12.0 - August 29, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fix diagnostic information when pushing or deploying source ([PR #1597](https://github.com/forcedotcom/salesforcedx-vscode/pull/1597)
+- Remove org browser flag ([PR #1598](https://github.com/forcedotcom/salesforcedx-vscode/pull/1598))
```

Comment:
@AnanyaJha, if the flag is removed because the feature is enabled by default, could we rephrase it as:
Enable Org Browser is the default setting, so the option to enable this feature is removed from the Settings editor.